src/software/ai/hl/stp/play/play.cpp

```diff
@@ -235,9 +235,7 @@ std::unique_ptr<TbotsProto::PrimitiveSet> Play::getPrimitivesFromTactic(
             *(motion_control.mutable_requested_destination()) =
                 *createPointProto(destination);
 
-            // TODO (#2610): we shouldn't have to check that the path has at least two
-            // knots, since the EnlsvgPathPlanner should always find paths with two points
-            if (path.has_value() && path.value().getKnots().size() >= 2)
+            if (path.has_value())
```

Comment:
we should be able to add a `CHECK (path.value().getKnots().size()>=2)` so make sure we don't have one point paths